honeybadgermpc/hbavss_light.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import asyncio
+from collections import defaultdict
+from pickle import dumps, loads
+from honeybadgermpc.betterpairing import ZR
+from honeybadgermpc.polynomial import polynomials_over
+from honeybadgermpc.poly_commit import PolyCommit
+from honeybadgermpc.symmetric_crypto import SymmetricCrypto
+from honeybadgermpc.exceptions import HoneyBadgerMPCError
+from honeybadgermpc.protocols.reliablebroadcast import reliablebroadcast
+
+
+class HbAVSSMessageType:
+    OK = "OK"
+
+
+class HbAvss(object):
```

Comment:
name this HbAvss light to make room for the batch version?

Reply:
Done.